### ISSUE TITLE
Submit to app Store

### ISSUE THREAD

**Issue by embassem** [aschuch/QRCode#39]
i guess you need to app a key to info.plist for Privacy - Media Library Usage Description
app Binary was invalid till i add  to my info.plist  the Error was missing this Key "NSPhotoLibraryUsageDescription"

please put a hint in readme file as it required from iOS 10

**Comment by aschuch** [comment]
I do not think this is directly related to this library, since it is not accessing the photo library directly?

**Comment by blochberger** [comment]
I vote for closing this issue. As @aschuch said, there is no need to access the photo library and therefore there is no need to set `NSPhotoLibraryUsageDescription` in the `Info.plist`.

**Comment by p-nicolaou** [comment]
@embassem do you use any other library that's accessing camera somehow? I faced the same issue (been rejected during app review) but that was due to another library [https://github.com/card-io/card.io-iOS-SDK](url)and the fact that is using `UIImagePickerController `. They only workaround is to add the **Key** in your plist file.
